ChelseasFieldJournal/src/ModMain.cs

```diff
@@ -1,4 +1,6 @@
-﻿using Ele.Utility;
+﻿
+using ChelseasFieldJournal;
+using Ele.Utility;
```

Comment:
Likely need to rename `Ele` to `Elephant`, given the refactor. Use the latest template version.